examples/exampleKotlinDslProject/build.gradle.kts

```diff
@@ -1,69 +1,59 @@
 import com.google.protobuf.gradle.*
-import org.gradle.kotlin.dsl.provider.gradleKotlinDslOf
 
-// A minimal example Java project that uses the protobuf plugin.
-// To build it:
-// $ ./gradlew clean build
 plugins {
-    java
-    idea
-    id("com.google.protobuf") version "0.8.8"
+  id("java")
+  id("idea")
+  id("com.google.protobuf") version "0.8.19"
 }
 
 repositories {
-    maven("https://plugins.gradle.org/m2/")
-}
-
-sourceSets{
-    create("sample"){
-        proto {
-            srcDir("src/sample/protobuf")
-        }
-    }
+  mavenCentral()
 }
 
 dependencies {
-    implementation("com.google.protobuf:protobuf-java:3.6.1")
```

Comment:
I didn't hide whitespace earlier, so I didn't notice this downgraded the versions significantly. I don't think we want that; if anything we'd upgrade them. I'm going to upgrade the `exampleProject` versions so they match the versions here.